apps/admin/frontend/src/components/export_election_package_modal_button.tsx

```diff
@@ -141,8 +140,8 @@ export function ExportElectionPackageModalButton(): JSX.Element {
       title = 'Election Package Saved';
       mainContent = (
         <P>
-          You may now eject the USB drive. Use the saved election package on
-          this USB drive to configure VxSuite components.
+          You may now eject the USB drive. Use the saved election package on the
+          USB drive to configure VxSuite components.
```

Comment:
<img width="540" alt="Screen Shot 2024-10-08 at 12 21 02 PM plus" src="https://github.com/user-attachments/assets/fae5bb72-f50e-48d9-bf67-1cff59c8b6e9">
<img width="545" alt="Screen Shot 2024-10-08 at 12 22 26 PM plus" src="https://github.com/user-attachments/assets/e19fc1dc-dc25-4443-9707-464b89a57dd4">

Reply:
Should all the buttons in these modals use the "USB Drive" language instead of "USB"?

Reply:
I'm voting to leave it as is to match the header button which should be short. Punting on the argument to fully remove the header button.